content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -88,187 +134,148 @@ following:
+(0,5]:0
+(5,10]:0
+(10,25]:0
+(25,50]:0
+(50,75]:0
+(75,100]:0
+(100,250]:0
+(250,500]:1
+(500,750]:0
+(750,1000]:0
+(1000,2500]:0
+(2500,5000]:0
+(5000,7500]:0
+(7500,10000]:0
+(10000,+Infinity]:0
+
+```
+
+</details>
+
+### Manual Metrics
 
-var MyActivitySource = new ActivitySource(serviceName);
+Next, add some manual metrics to the app. This will initialize a
+[Meter](/docs/concepts/signals/metrics) to create a counter in code.
 
+```csharp
 var builder = WebApplication.CreateBuilder(args);
 
-var appResourceBuilder = ResourceBuilder.CreateDefault()
-    .AddService(serviceName: serviceName, serviceVersion: serviceVersion);
+// .. other setup
 
-builder.Services.AddOpenTelemetryTracing(tracerProviderBuilder =>
-{
-    tracerProviderBuilder
-        .AddConsoleExporter()
-        .AddSource(MyActivitySource.Name)
-        .SetResourceBuilder(appResourceBuilder)
-        .AddHttpClientInstrumentation()
-        .AddAspNetCoreInstrumentation()
-        .AddSqlClientInstrumentation();
-});
-
-var meter = new Meter(serviceName);
-var counter = meter.CreateCounter<long>("app.request-counter");
-builder.Services.AddOpenTelemetryMetrics(metricProviderBuilder =>
+builder.Services.AddOpenTelemetry()
+    .WithTracing(/*  .. tracing setup */ )
+    .WithMetrics(metricsProviderBuilder =>
+        metricsProviderBuilder
+            .AddMeter(DiagnosticsConfig.Meter.Name)
+			// .. more metrics
+             );
+
+public static class DiagnosticsConfig
 {
-    metricProviderBuilder
-        .AddConsoleExporter()
-        .AddMeter(meter.Name)
-        .SetResourceBuilder(appResourceBuilder)
-        .AddAspNetCoreInstrumentation()
-        .AddHttpClientInstrumentation();
-});
+    public const string ServiceName = "MyService";
 
-var app = builder.Build();
+    // .. other config
 
-app.MapGet("/hello", () =>
-{
-    // Track work inside of the request
-    using var activity = MyActivitySource.StartActivity("SayHello");
-    activity?.SetTag("foo", 1);
-    activity?.SetTag("bar", "Hello, World!");
-    activity?.SetTag("baz", new int[] { 1, 2, 3 });
+    public static Meter Meter = new(ServiceName);
+    public static Counter<long> RequestCounter =
+        Meter.CreateCounter<long>("Request Counter");
+}
 
-    // Up a counter for each request
-    counter.Add(1);
+```
+
+Now we can increment the counter in our `Index` action.
+
+```csharp
+    public IActionResult Index()
+    {
+        // do other stuff
 
-    return "Hello, World!";
-});
+        DiagnosticsConfig.RequestCounter.Add(1,
+            new("Action", nameof(Index)),
+            new("Controller", nameof(HomeController)));
 
-app.Run();
+        return View();
+    }
 ```
 
-The output will be similar as with tracing, but now includes the request counter
-and periodically exports a histogram of request times for each route.
+You'll notice here that we're also adding labels to our request counter that
+distinguish it from other requests. You should now see an output like this.
```

Comment:
```suggestion
You'll notice here that we're also adding labels to our request counter that
distinguishes it from other requests. You should now see an output like this.
```

Reply:
Replace term "labels" with either "attributes" or "tags" here.

Label was previously a term used in the spec to describe dimensions on metrics - this term was changed in the spec to attribute. Though, of course, .NET is a 🦄 and uses the term tag... so 🤷.

Reply:
Fixed